svm/src/transaction_processor.rs

```diff
@@ -373,28 +386,7 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
             }
 
             program_cache_for_tx_batch
-        });
-        execute_timings
-            .saturating_add_in_place(ExecuteTimingType::ProgramCacheUs, program_cache_us);
-
-        // Determine a capacity for the internal account cache. This
-        // over-allocates but avoids ever reallocating, and spares us from
-        // deduplicating the account keys lists.
-        let account_keys_in_batch = sanitized_txs.iter().map(|tx| tx.account_keys().len()).sum();
-
-        // Create the account loader, which wraps all external account fetching.
-        let mut account_loader = AccountLoader::new_with_loaded_accounts_capacity(
-            config.account_overrides,
-            callbacks,
-            &environment.feature_set,
-            account_keys_in_batch,
-        );
-
-        // Create the transaction balance collector if recording is enabled.
-        let mut balance_collector = config
-            .recording_config
-            .enable_transaction_balance_recording
-            .then(|| BalanceCollector::new_with_transaction_count(sanitized_txs.len()));
```

Comment:
note this code did not actually change, the diff algo is just anchoring the new replenish call to the existing one